apps/web/src/pages/settings/user-profile-page/UserProfilePasswordEmailVerificationSection.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, IconOutlineForwardToInbox, IconOutlineMarkunreadMailbox } from '@novu/design-system';
+import { css, cx } from '../../../styled-system/css';
+import { HStack, Stack } from '../../../styled-system/jsx';
+import { text } from '../../../styled-system/recipes';
+import { IUserProfilePasswordEmailVerificationProps } from './UserProfilePasswordSidebar.shared';
+
+export const UserProfilePasswordEmailVerificationSection: React.FC<IUserProfilePasswordEmailVerificationProps> = ({
+  email,
+  sendVerificationEmail,
+  countdownTimerSeconds,
+}) => {
+  const isButtonDisabled = countdownTimerSeconds > 0;
+
+  return (
+    <Stack direction={'column'} gap={'200'} mt="150">
+      <HStack gap="100">
+        {/* TODO: this is a different Icon from designs since the one from React Icons doesn't match the name */}
+        <IconOutlineMarkunreadMailbox size="40" className={css({ flexShrink: 0 })} />
+        <p className={cx(text(), css({ color: 'typography.text.secondary' }))}>
+          We have just sent a verification link to your email address{' '}
+          <span className={text({ variant: 'main' })}>{email}</span>.<br />
+          Please verify your email address to proceed with setting a password.
+        </p>
+      </HStack>
+      <HStack gap={'200'} justifyContent="flex-end">
+        <span className={text()}>
+          Didn't get the link? Resend in{' '}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** When the timer stops, `Resend in 0 seconds` looks odd. Maybe we can just write `resend` or `resend now` when the timer is 0‏

Reply:
Yeah, we didn't have designs that handled this, and agreed. However, I tried it, and it causes the text to jump which I don't think looks great either. We can get more input in the UX hand-off